portfolio/src/main/webapp/style.css

```diff
@@ -316,3 +316,65 @@ section .courses ul li span {
   margin: 5px;
 }
 
+/* -------------- Project Hover Cards --------------*/
+
+.container {
+  position: relative;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items:center;
+  flex-wrap: wrap;
+  padding: 30px;
+}
+
+.container .card {
+  max-width: 300px;
+  height: 300px;
+  position: relative;
+  background: #fff;
+  margin: 30px 10px;
+  padding: 25px 15px;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 0 5px 20px rgba(0,0,0,.3);
+  transition: 0.5s ease-in-out;
+  border-radius: 5px;
+}
+
+.container .card:hover {
+  height: 550px;
```

Comment:
So the main difficulty with hardcoding these heights is that you need to ensure your text is within a certain length to make it look right. I'd recommend that instead you make the div contain its entire content, with the paragraph defaulting to display: none (which makes it occupy 0 height). Then on hover, remove the display: none on the paragraph, which implicitly adds height to the card. Let me know if you have questions on how this works.